tests/functional/aws-node-sdk/test/bucket/putCors.js

```diff
@@ -99,6 +99,12 @@ describe('PUT bucket cors', () => {
             _testPutBucketCors(testCors, 400, 'InvalidRequest', done);
         });
 
+        it('should return InvalidRequest for lowercase value for ' +
```

Comment:
Here I'm not comparing values from putting and getting the CORS configuration. AWS actually rejects lowercase values for AllowedMethods when putting the CORS configuration in the first place, and that's what this test shows.

Can add another test that shows case is preserved for AllowedMethods when getting CORS configuration.